Gruntfile.js

```diff
@@ -137,11 +140,29 @@ module.exports = function(grunt) {
 					'lib/commons/index.js',
 					'lib/commons/*/index.js',
 					'lib/commons/**/*.js',
+
+					// directories we've converted to ES Modules
+					'!lib/commons/utils/*.js',
+
+					// output of webpack directories
+					'tmp/commons/**/*.js',
+
 					'lib/commons/outro.stub'
 				],
 				dest: 'tmp/commons.js'
 			}
 		},
+		webpack: {
+			commonsUtils: {
+				devtool: 'source-map',
```

Comment:
Do we do anything with the source map generated here? I think we should either be using an "inline" source map or none at all (unless there's some build magic elsewhere that I'm not aware of).

Reply:
I'm not sure? I just read it prevents webpack from turning it into a string and evaling it.

Reply:
I am **strongly** against copy/pasting Webpack configuration without having a full understanding of what it's doing. If you want help with Webpack stuff, please LMK (send a Slack message).

Per [the `devtool` docs](https://webpack.js.org/configuration/devtool/):

> source-map - A full SourceMap is emitted as a separate file. It adds a reference comment to the bundle so development tools know where to find it.

This means Webpack will create a source-map in a separate file for the bundle you're creating here (as well as every other bundle created). Since we're not going to do anything with this source map, I do not believe we want this. It's going to slow down the build and provide no benefit.

If we want source maps for development purposes, we should use one Webpack's "inline" options (so there's no extra build step to include the map in our bundle(s)). However, I do not recall source maps being discussed in your proposal for this change, so I think we can safely leave them out for now.